app/Console/ConsoleServiceProvider.php

```diff
@@ -439,13 +441,22 @@ function () use ($container) {
 
         $container->addShared(
             'console.place:update-locality',
-            fn () => new UpdateLocality(
+            fn () => new UpdatePlaceLocality(
                 $container->get('event_command_bus'),
                 $container->get(PlacesSapi3SearchService::class),
                 $container->get('place_jsonld_repository')
             )
         );
 
+        $container->addShared(
+            'console.organizer:update-locality',
+            fn () => new UpdateOrganizerLocality(
+                $container->get('event_command_bus'),
+                $container->get(OrganizersSapi3SearchService::class),
+                $container->get('place_jsonld_repository')
```

Comment:
This seems to be the wrong repository. Were you able to test this new CLI command?